devicetypes/stelpro/stelpro-ki-zigbee-thermostat.src/stelpro-ki-zigbee-thermostat.groovy

```diff
@@ -315,10 +315,6 @@ def parse(String description) {
 				} else {
 					map.value = "heating"
 				}
-
-				if (settings.heatdetails == "No") {
-					map.displayed = false
```

Comment:
This should only affect the event history, and in Classic. If this is actually preventing events from reaching the event pipeline, then there is a bug in data management.

See https://community.smartthings.com/t/stelpro-releases-the-first-z-wave-plus-line-voltage-thermostat-the-stzw402-stelpro-ki-thermostat/39278/179 for a reference to the purpose of this.

Reply:
Based on the live logs I was seeing, it wasn't getting switched anyway (but the events were). I was just throwing stuff at the wall.

Reply:
My guess is it's actually a ui metadata pointing at the wrong href.

Reply:
Turns out that the device was being configured to not report changes more frequently than 5 minutes, as well as the cadence of the duty cycles and the code that handles passing off the Zigbee messages. The ticket(s) include details.